helm_deploy/hmpps-sentence-plan-ui/values.yaml

```
@@ -77,3 +77,6 @@ generic-service:
 generic-prometheus-alerts:
   targetApplication: hmpps-sentence-plan-ui
   alertSeverity: probation-integration-notifications
+
+# Alert for business hours only - 7am to 6pm UTC (8am to 7pm BST) on week days (Monday to Friday)
+businessHoursOnly: true
```

I think you need to indent this, so it's under `generic-prometheus-alerts`
```suggestion

  # Alert for business hours only - 7am to 6pm UTC (8am to 7pm BST) on week days (Monday to Friday)
  businessHoursOnly: true
```